R/verb-fill.R

```diff
@@ -64,6 +68,18 @@ fill.tbl_lazy <- function(.data, ..., .direction = c("down", "up")) {
   )
 }
 
+swap_order_direction <- function(x) {
+  if (is_quosure(x)) {
+    x <- quo_get_expr(x)
+  }
+
+  if (is_call(x, "desc")) {
```

Comment:
```suggestion
  if (is_call(x, "desc", n = 1)) {
```

Not very important, but protects against a weirder error if you have `desc()`.